tests/test_connections.py

```diff
@@ -33,27 +38,36 @@ def test_default_connection_is_returned_by_default():
 
 
 def test_get_connection_created_connection_if_needed():
-    c = connections.Connections()
-    c.configure(default={"hosts": ["es.com"]}, local={"hosts": ["localhost"]})
+    c = connections.Connections(elasticsearch_class=DummyElasticsearch)
```

Comment:
Those tests are about `elasticsearch_dsl.connections`, not about how we store hosts in `Elasticsearch`. In elasticsearch-py 8.x, the hosts are technically available, but they're inside a deep hierarchy of classes, so using a dummy class avoids depending on those internals.